softwarereview_policies.Rmd

```diff
@@ -157,6 +157,120 @@ Despite our best efforts to support contributed software, errors are the respons
 
 In the unlikely scenario that a contributor of a package requests removal of their package from the suite, we retain the right to maintain a version of the package in our suite for archival purposes.
 
+## Ethics, Data Privacy and Human Subjects Research
+
+rOpenSci tools are used for a variety of purposes, but our focus is on
+tools for research. We expect that tools will enable ethical use by research
+practitioners, who are obligated to adhere to ethical codes such [Declaration of
+Helsinki](https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/)
+and [The Belmont
+Report](https://www.hhs.gov/ohrp/regulations-and-policy/belmont-report/index.html).
+Researchers bear responsibility for their use of software, but software
+developers must consider the ethical use of their products, and developers
+themselves adhere to ethical codes for computer professionals such as those
+expressed by [IEEE](https://www.computer.org/education/code-of-ethics) and
+[ACM](https://ethics.acm.org/). rOpenSci contributors often play both roles.
+
+We ask that software developers place themselves in researchers' role and
+consider the requirements of an ethical workflow using authors' software.
+Given the variation and degree of flux of ethical approaches for Internet-based
+analyses, judgement calls rather than recipes are required. The [Ethical
+Guidelines of The Association of Internet Researchers](https://aoir.org/ethics/)
+provides a robust framework and we encourage authors, editors, and reviewers to use
+this in evaluating their work. In general, adherence to legal or regulatory
+minimum requirements may not be sufficient, though these (e.g.,
+[GPDR](https://ico-gdpr.com/)), may be relevant. Package authors should direct
+users to relevant resources for the ethical use of the software.
+
+Some packages, due to the nature of data they handle, may be determined by editors to require enhanced scrutiny. For these, editors may require additional (or reduced) functionality, and robust
+documentation, defaults, and warnings to direct users to relevant ethical
+practices. The following topics may merit enhanced scrutiny:
+
+-   *Vulnerable populations*: Authors of packages and workflows that deal with
+    information related to vulnerable populations bear responsibility to protect
+    them from likely harms.
+
+-   *Personally identifiable or sensitive data*: The release of personally
+    identifiable or sensitive data is potentially harmful, and data may be
+    re-identifiable even if anonymized. Packages that access or manage data
+    sources that include personally-identifiable information should include
+    functionality, documentation, and demonstrations of workflows to minimize
```

Comment:
we don't know of any existing example of such documentation, right?

Reply:
Not yet!